### PR TITLE
2022 07 22 public feedback

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/Arcueid/Day 7/07_00_ARC07_3F.txt [hunk 1]
```diff
@@ -90,7 +90,8 @@ I'll circle back on her physical condition later.
 -- Page 342, Offset 17425.
 -- さっきのヤツは吸血鬼の手足になってる死者だな？
 // Maybe see if we can come up with a more _meaningful_ translation of 手足.
-That guy from earlier, he was one of those Dead that act as a vampire's worker drones, wasn't he?
+// or just worker
+That guy from earlier, he was one of those Dead that act as a vampire's worker ant, wasn't he?
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3F.txt:94]: acts as a vampire's worker ant. or
act as a vampire's worker ants.

is this the only instance? I know there are some other instances of 働き蟻

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 7/07_00_ARC07_3F.txt (hunk 1 above)
Comment (Volvo-hackangel) [reply script/Arcueid/Day 7/07_00_ARC07_3F.txt:94]: ok let me check

Comment (Volvo-hackangel) [reply script/Arcueid/Day 7/07_00_ARC07_3F.txt:94]: there's a bunch, they're all just translated to just worker, should I follow suit?
https://github.com/Tsukihimates/Tsukihime-Translation/blob/95e62b493d32eaab21ce670548ba6410aa2f06e5/script/Arcueid/Day%207/07_00_ARC07_3_1_1.txt#L80
https://github.com/Tsukihimates/Tsukihime-Translation/blob/95e62b493d32eaab21ce670548ba6410aa2f06e5/script/Arcueid/Day%207/07_00_ARC07_3_1_1.txt#L84
https://github.com/Tsukihimates/Tsukihime-Translation/blob/95e62b493d32eaab21ce670548ba6410aa2f06e5/script/Arcueid/Day%207/07_00_ARC07_3_1_1.txt#L451
https://github.com/Tsukihimates/Tsukihime-Translation/blob/95e62b493d32eaab21ce670548ba6410aa2f06e5/script/Arcueid/Day%208/08_00_ARC08_6B.txt#L1297

Comment (Petrikow) [reply script/Arcueid/Day 7/07_00_ARC07_3F.txt:94]: Any reason you are changing this?

**Review by rschlaikjer — APPROVED** [review]

**Pull request merged**